server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -181,7 +185,36 @@ private FormTag buildProgramForm(
             .setValue(DisplayMode.SELECT_TI.getValue())
             .setChecked(displayMode.equals(DisplayMode.SELECT_TI.getValue()))
             .getRadioTag(),
-        showTiSelectionList(selectedTi, displayMode.equals(DisplayMode.SELECT_TI.getValue())),
+        showTiSelectionList(selectedTi, displayMode.equals(DisplayMode.SELECT_TI.getValue())));
+
+    formTag.with(
+        // TODO(#2618): Consider using helpers for grouping related radio controls.
+        fieldset()
+            .with(
+                legend("Program eligibility gating")
+                    .withClass(BaseStyles.INPUT_LABEL)
+                    .with(ViewUtils.requiredQuestionIndicator())
+                    .with(p("(Not applicable if this program is the pre-screener)")),
```

Comment:
suggestion (blocking): This text might be confusing since some CEs might not be using pre-screeners. I suggest conditionally showing this text if the prescreener feature flag is enabled and/or only showing this section when the pre-screener button is NOT checked.

Reply:
Oh that's a great callout! Updated to only show if the flag is enabled.